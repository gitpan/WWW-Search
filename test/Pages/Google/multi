http://www.google.com/search?num=100&q=LINUX+card+games
http://www.google.com/search?q=LINUX+card+games&num=100&start=100&sa=N
http://www.google.com/search?q=LINUX+card+games&num=100&start=1000&sa=N><IMG SRC=/nav.gif alt=1001-1100 border=0><br><font color=purple face=arial,helvetica size=-2>11</A></td><td><A HREF=/search?q=LINUX+card+games&num=100&start=200&sa=N
http://www.google.com/search?q=LINUX+card+games&num=100&start=1000&sa=N><IMG SRC=/nav.gif alt=1001-1100 border=0><br><font color=purple face=arial,helvetica size=-2>11</A></td><td><A HREF=/search?q=LINUX+card+games&num=100&start=1100&sa=N><IMG SRC=/nav.gif alt=1101-1200 border=0><br><font color=purple face=arial,helvetica size=-2>12</A></td><td><A HREF=/search?q=LINUX+card+games&num=100&start=300&sa=N
