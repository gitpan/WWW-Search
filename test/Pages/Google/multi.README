This set of test-result pages was created on Mon Aug 16 16:26:27 1999
