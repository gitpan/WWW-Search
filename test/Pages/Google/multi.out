http://www.linuxapps.com/
http://www.cdromshop.com/cdshop/desc/p.020626704809.html
http://www.cdromshop.com/cdshop/desc/p.020626707565.html
http://www.cdromshop.com/cdshop/desc/p.076930995181.html
http://www.cdromshop.com/cdshop/desc/p.747851012058.html
http://www.cdromshop.com/cdshop/w95/cat18.html
http://www.cdromshop.com/cdshop/desc/p.705381036005.html
http://www.cdromshop.com/cdshop/desc/p.790561501510.html
http://www.linuxrepublic.com/
http://rpmfind.net/linux/RPM/VByName.html
http://rpmfind.net/linux/RPM/Games.html
http://rufus.w3.org/linux/RPM/X11_Games_Strategy.html
http://rufus.w3.org/linux/RPM/VByName.html
http://www.linuxjournal.com/bot/
http://www.gamestats.com/todaysnews.html
http://phoenix.evansville.edu/~raistlin/linux/
http://www.linuxlinks.com/Software/Games/Board_and_Card_Games/index.shtml
http://www.linuxlinks.com/Software/Games/
http://www.solitairecentral.com/links.html
http://computing.net/windows95/wwwboard/wwwboard.html
http://computing.net/unix/wwwboard/wwwboard.html
http://i-want-a-website.com/about-linux/mar99.shtml
http://i-want-a-website.com/about-linux/articles.shtml
http://www.yahoo.com/
http://www1.zdnet.com/zdhelp/stories/main/0,5594,2225255-10,00.html
http://www8.zdnet.com/products/stories/reviews/0,4161,374896,00.html
http://www8.zdnet.com/smallbusiness/stories/general/0,5821,333103,00.html
http://www.silug.org/LDP/HOWTO/Sound-HOWTO-6.html
http://www.silug.org/LDP/HOWTO/Sound-HOWTO-4.html
http://www.io.com/help/linux/Sound-HOWTO-6.html
http://www.io.com/help/linux/Sound-HOWTO-4.html
http://www.tdl.com/~netex/html/Sound-HOWTO-6.html
http://www.tdl.com/~netex/html/Sound-HOWTO-4.html
http://www.itlibrary.com/reference/library/0672309629/rhl60.htm
http://www.itlibrary.com/reference/library/0672310120/html/ch65.htm
http://www.itlibrary.com/reference/library/0672308509/lsg10.htm
http://answersleuth.com/topics/Games.shtml
http://www.slashdot.org/askslashdot/98/11/29/1349234.shtml
http://www.hensa.ac.uk/linux/rpm2html/X11_Games_Strategy.html
http://www.hensa.ac.uk/linux/rpm2html/VByName.html
http://www.hensa.ac.uk/linux/rpm2html/Games.html
http://www.herbison.com/herbison/bj_spades.html
http://slashdot.org/askslashdot/99/04/07/2242222.shtml
http://www.zdnet.com/products/stories/reviews/0,4161,374896,00.html
http://www.zdnet.com/talkback/201_22123_92623.html
http://www.zdnet.com/swlib/specials/linux/stampede.html
http://www.zdnet.com/zdhelp/howto_help/monitor/monitor_7.html
http://www.zdnet.com/talkback/201_22123_95164.html
http://www.zdnet.com/swlib/specials/linux/suse.html
http://www.zdnet.com/products/stories/reviews/0,4161,371369,00.html
http://www.zdnet.com/pcmag/pctech/download/shareware.html
http://www.zdnet.com/swlib/pilotsoftware/prespick/1098/ppp1098.html
http://www.linuxberg.com/
http://www.billsgames.com/contact/
http://www.billsgames.com/aboutus/
http://www.gy.com/www/
http://www.mysharewarepage.com/gamepatch2.htm
http://www.gamingplace.com/cardgames.htm
http://www.cdrom.com/titles/games/giga5.phtml
http://www.mlug.org/links.html
http://www.jumbo.com/pages/games/dos/
http://www.freest.com/hotlink.htm
http://jumbo.com/pages/games/dos/
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/multiplayer/!INDEX.short.html
http://metalab.unc.edu/pub/Linux/games/solitaires/!INDEX.html
http://eunuchs.org/linux/multimedia/mmedia_content.html
http://www.bigmall.com/
http://groups.mirabilis.com/category.html
http://oak.ece.ul.ie/~griffini/linux.html
http://www.linux.org.il/linux-il-faq.html
http://billsworkshop.com/hotnews.html
http://www.indstate.edu/CU-SeeMe/devl_archives/oct_95/0671.html
http://www.eonova.com/dist/0.90/pkgindex/X11_Games.html
http://webcrawler.com/people_and_chat/games_and_chat/online_games/
http://www.independence.seul.org/software/vga_cardgames.html
http://www.icq.com/icqlist/
http://www.icq.com/members/
http://www.icq.com/icqchat/
http://weber.u.washington.edu/~lauridj/links.html
http://www.gueststar.demon.nl/linuxlinks.html
http://www.gueststar.demon.nl/linuxlinks-nojs.html
http://magellan.excite.com/games/online_games
http://www.flfsoft.com/100_links/
http://www.cbf.ca/GBL/gblSOFT/
http://www.xlr8yourmac.com/G3-ZONE/yosemite/
http://www.linux.victoria.bc.ca/linux/links/Games/
http://www.os2bbs.com/Download/games.html
http://www.os2bbs.com/download/games.html
http://www.byte.com/art/9602/sec11/art7.htm
http://bahamut.mm.t.u-tokyo.ac.jp/~iwai/awedrv/awedrv-faq.html
http://www.zdwebopedia.com/Networks/network_interface_card_NIC.html
http://www.comune.lucca.it/ASS_VOL/MOUSE/lugames.html
http://www.geocities.com/SiliconValley/Lakes/2628/t430cds.html
http://www.student.wau.nl/~olivier/midi/soundcard.htm
http://www.cs.cmu.edu/afs/cs.cmu.edu/user/tilt/public_html/games.html
http://www.xnet.com/~blatura/linapp1.html
http://www.xnet.com/~blatura/text1.html
http://www.software.fsn.net/produkt.html
http://www.software.fsn.net/spill.html
http://www.tiklsoft.com/programs.html
http://www.3dfiles.com/a3dcentral/faq.shtml
http://www.mindspring.com/~clagett/bill/links/
http://www.zdnet.com/yil/stories/doubledl/0,5207,2160215,00.html
http://www.zdnet.com/zdhelp/stories/main/0,5594,2225255-10,00.html
http://www.zdnet.com/swlib/games.html
http://www.zdnet.com/products/stories/reviews/0,4161,309166,00.html
http://www.zdnet.com/products/stories/reviews/0,4161,388749,00.html
http://www.linuxmall.com/
http://www.linuxmall.com/Allprod/00868.html
http://www.linuxmall.com/sofps.html
http://www.ao.com/~regan/home.html
http://members.tripod.com/jeffchiu/linux-i3200.html
http://www.pdacentral.com/psionking/communications5.htm
http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Software/Games/
http://www.itknowledge.com/reference/library/0672313723/ewtoc.html
http://www.itknowledge.com/reference/library/0672309084/ewtoc.html
http://www.itknowledge.com/reference/library/1558285660/book-index.html
http://www.itknowledge.com/reference/library/1562764497/ewtoc.html
http://www.linux.ie/LWR/
http://sg.yahoo.com/Business_and_Economy/Companies/Computers/Software/Games/
http://www.eonova.com/dist/0.90/pkgindex/Games.html
http://www.gweep.net/~jer/bookmark.html
http://www.linpro.no/wine/development/WineAndCicaGames.002.html
http://www.fokus.gmd.de/linux/linux-softw-sys.html
http://rufus.w3.org/linux/RPM/conectiva/3.0/i386/ByDate.html
http://www.forevermail.com/domain-list.html
http://english.hk.yahoo.com/Business_and_Economy/Companies/Computers/Software/Games/
http://www.cs.helsinki.fi/~mjrauhal/linux/cola.archive/cola-1999-02.html
http://asia.yahoo.com/Business_and_Economy/Companies/Computers/Software/Games/
http://www.chicken.org/brett/bookmarks.html
http://www.gaijin.com/X/linksApps.html
http://www.y2kinfo.com/directories/pages/category.html
http://www.developer.com/directories/pages/category.html
http://www.aunz.yahoo.com/Business_and_Economy/Companies/Computers/Software/Games/
http://linuxcentral.com/products/soft/linuxgames/
http://www.cdromshop.com/cdshop/desc/p.020626704960.html
http://www.cdromshop.com/cdshop/desc/p.081656087126.html
http://www.cdromshop.com/cdshop/desc/p.081656086129.html
http://www.cdromshop.com/cdshop/desc/p.020626702553.html
http://www.cdromshop.com/cdshop/desc/p.020626707022.html
http://www.slashdot.org/askslashdot/98/12/19/1719241.shtml
http://www.slashdot.org/askslashdot/9807140932243.shtml
http://www.slashdot.org/askslashdot/980803/1322257.shtml
http://www.slashdot.org/askslashdot/980807/1046246.shtml
http://www.linuxgames.com/msgboard/qa/
http://privat.riksnett.no/~bthoem/warpcentral/software!_share-_and_freeware.html
http://ca.yahoo.com/Business_and_Economy/Companies/Computers/Software/Games/
http://www.yahoo.co.uk/Business_and_Economy/Companies/Computers/Software/Games/
http://www.wagnerweb.org/doug/linux/
http://www.pda-archives.com/pilot/pilotmap.htm
http://www.dma.org/~pfeiferb/DrBob_ndx.html
http://www.greatshop.com/cdrom/home.html
http://www.herbison.com/herbison/bj_games.html
ftp://ftp.kiae.su/pub/new9809/
http://www.oglibrary.com/rpgef.html
http://www.oglibrary.com/actionab.html
http://www.oglibrary.com/rpgab.html
http://magellan.excite.com/people_and_chat/games_and_chat/online_games
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/solitaires/!INDEX.short.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/solitaires/!INDEX
http://metalab.unc.edu/pub/Linux/games/solitaires/!INDEX.short.html
http://metalab.unc.edu/pub/Linux/games/solitaires/!INDEX
http://www.tux.org/hypermail/linux-net/1998-Nov/
http://www.sharewarejunkies.com/whatsnew.htm
http://www.mmwhse.com/lists/catalog.htm
http://www.mmwhse.com/lists/lists.htm
http://www.pbs.mcp.com/ebooks/1562764497/
http://www.swregnet.com/byalpha.htm
http://www.swregnet.com/category.htm
http://www.swregnet.com/contents.htm
http://www1.zdnet.com/zdhelp/howto_help/dvd/dvd_5.html
http://aldev.8m.com/
http://www.cdrom.com/titles/linux/linux_games.phtml
http://www.image.dk/~flywheel/os2/sa/os2links.html
http://stuffall.com/software/chat.shtml
http://www.independence.seul.org/software/xpat2.html
http://homepages.enterprise.net/colinp/links.shtml
http://slashdot.org/articles/99/04/15/0242208_F.shtml
http://www.linuxgazette.com/issue33/ayers2.html
http://theyellowpages.com/g.htm
http://www.ai.univie.ac.at/~juffi/links.html
http://www.zdwebopedia.com/Hardware/smart_card.html
http://www.zdwebopedia.com/Multimedia/sound_card.html
http://www.zdwebopedia.com/Portable_Computing/PC_card.html
http://www.linuxlinks.com/New/
http://www.surfpoint.com/Games--Fantasy_Role_Playing-1.html
http://www.landfield.com/usenet/news.announce.newgroups/New_Usenet_Groups/19941216
http://www.landfield.com/usenet/news.announce.newgroups/New_Usenet_Groups/19941213
http://info-s.com/sub.html
http://daft.com/~torin/bookfull.html
http://thecity.sfsu.edu/LDP/links.html
http://termiitti.akumiitti.fi/~orava/links.html
http://www.rediff.com/computer/1999/mar/17free.htm
http://www.haystack.edu/llnews.html
http://www.topology.org/linux.html
http://i-want-a-website.com/about-linux/jul98.shtml
http://i-want-a-website.com/about-linux/dec98.shtml
http://libweb.sonoma.edu/mike/fujitsu/
http://linux.umbc.edu/mirrors/newbie-admin-faq/FAQ.htm
http://www.jps.net/coolpages/
http://www.xnet.com/~blatura/linapp5.html
http://www.gamestats.com/archive/100498todaysnews.html
http://www.cdromshop.com/cdshop/desc/p.3348542050945.html
http://www.cdromshop.com/cdshop/ibm/cat18.html
http://www.software4u.com/SOFTWARE.htm
http://www.smartbooks.com/bw708linuxconfgchp.htm
http://www.gameslog.com/voodoo-guru/
http://www.memepool.com/Author/goboro/
http://www.developer.com/directories/pages/dir.java.game.multiplayer.13.html
http://www.hongik.com/linux/textonly.html
http://www.hongik.com/linux/linapp1.html
http://www-personal.engin.umich.edu/~jgotts/linux.html
http://rufus.w3.org/linux/RPM/X11_Games.html
http://www.incatec.com/shareware.html
http://www.slashdot.org/askslashdot/9806202127214.shtml
http://terra.mpikg-teltow.mpg.de/~burger/T400CDT-Linux.html
http://niteowl.userfriendly.net/linux/RPM/Pacific_HiTech,_Inc.html
http://www.maschek.com/
http://www.itknowledge.com/reference/library/078971132x/appendix-c.html
http://www.itknowledge.com/reference/library/0672313723/book-index.html
http://www.itknowledge.com/reference/library/1558285660/ewtoc.html
http://scrye.com/~loki/bookmarks.html
http://metalab.unc.edu/pub/Linux/docs/HOWTO/Sound-HOWTO
http://metalab.unc.edu/pub/Linux/NEW.logs/NEW.1997-03-04.html
http://www.zdnet.com/swlib/prespick/0798/pres0798.html
http://www.zdnet.com/sr/stories/issue/0,4537,387506,00.html
http://www.palmcentral.com/Development(SampleCode).2.html
http://www.fastlane.net/~russellm/gaminglinks.html
http://www.fastlane.net/~russellm/
http://www.angelfire.com/ca/talvola/Freesw.html
http://www.macwindows.com/newsarch.html
http://www.eio.com/repairfaq/REPAIR/F_ffmon.html
http://www.independence.seul.org/software/index.RedHat.html
http://www.linux-kheops.com/pub/miroir/linapps/textonly.shtml
http://www.indelible-blue.com/ibapps/products.nsf/AE
http://www.lib.uchicago.edu/~tao/home.html
http://www8.zdnet.com/zdhelp/stories/main/0,5594,2225255-10,00.html
http://www.sonador.com/News/list.html
http://www.calderasystems.com/tech-ref/linuxapps/textonly.html
http://www.umr.edu/~cmaloney/bookmark.html
http://ca.us.mirrors.freshmeat.net/appindex/x11/games.html
http://www.nmt.edu/~mikeshaw/
http://pages.prodigy.com/MikeShaw/
http://www.cs.berkeley.edu/~laura/hotlist.html
http://biz.yahoo.com/n/l/logiy.html
http://members.xoom.com/InfoSysSec/apps.html
http://members.xoom.com/stephen/specialized.html
http://slashdot.org/articles/99/04/05/1831237.shtml
http://freshmeat.iweb.net.au/appindex/x11/games.html
ftp://ftp.nectec.or.th/pub/mirrors/linux/NEW.logs/NEW.1997-03-04.html
http://www.rge.com/pub/systems/linux/sunsite/NEW.logs/NEW.1997-03-04.html
http://www.eonova.com/dist/0.90/pkgindex/ByDate.html
http://rulhm1.leidenuniv.nl/~boris/linux_list.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/NEW.logs/NEW.1997-03-04.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/multiplayer/!INDEX.html
http://core.freshmeat.net/appindex/x11/games.html
http://www.webnix.com/linux/lg/issue31/issue31.html
http://linuxcentral.com/linux/lg/issue31/issue31.html
http://www.zdwebopedia.com/Hardware/card.html
http://www.rose-hulman.edu/~froydnj/bookmarks.html
http://www.hensa.ac.uk/linux/rpm2html/Red_Hat_Linux_Colgate.html
http://www.grot.com/zoomer/zoomer-list/ogo-4Q1997/0081.html
http://www.delorie.com/store/ace/docs/toolkit.html
http://www.linuxjournal.com/bg/vertical.html
http://www.nwc.com/unixworld/tutorial/012/012.html
http://starvector.com/cd/
http://www.publiccom.com/web/mikey/lifebooklinux.html
http://shareware.gamespot.com/adjuncts/000hue.html
http://www.maurine.com/driver.htm
http://home.sol.no/~sverrehu/pub-unix/
http://www.cs.helsinki.fi/~mjrauhal/linux/cola.archive/cola-1998-06.html
http://go.muzi.net/Business/Companies/Computers/Software/Games/
http://www.penguincomputing.com/LDP/LDP/LG/issue35/issue35.html
http://www.penguincomputing.com/LDP/LDP/LG/issue31/issue31.html
http://isunix.it.ilstu.edu/resources/lg/issue35/issue35.html
http://isunix.it.ilstu.edu/resources/lg/issue31/issue31.html
http://www.chez.com/mmcbridge/Divers/hotlist.html
http://www.linuxgazette.com/issue31/issue31.html
ftp://ftp.the-gc.net/lg/issue31/issue31.html
http://www.ucalgary.ca/UofC/faculties/GNST/theses/avis/bibliog.html
ftp://ftp.patoche.org/lg/issue31/issue31.html
http://www.linuxhq.com/lg/issue31/issue31.html
http://www.minet.net/docs/lg/issue31/issue31.html
http://www.freesource.org/lg/issue31/issue31.html
http://www.lg.clight.net/issue31/issue31.html
http://www.deva.net/lg/issue31/issue31.html
http://www.patoche.org/lg/issue31/issue31.html
http://lgazette.linuxfr.org/issue31/issue31.html
http://lg.iafrica.com/issue31/issue31.html
http://www.uni-paderborn.de/Linux/mdw/linux.html
http://www.linuxgames.com/quake/Quake-HOWTO-2.html
http://wa.us.mirrors.freshmeat.net/appindex/x11/games.html
http://www.zdnet.com/zdnn/stories/zdnn_display/0,3440,2118577,00.html
http://www.zdnet.com/zdhelp/filters/games/
http://www.oir.ucf.edu/LDP/links.html
http://www.itlibrary.com/reference/library/0672308509/lsg11.htm
http://www.debian.org/lintian/reports/packages_2.html
http://www.debian.org/lintian/reports/depcheck.html
http://www.ncf.carleton.ca/auto-gen/ip-index/t.html
http://www.linuxmall.com/sofmi.html
http://www.bcpl.net/~wnidiffe/fun.html
http://searchlinux.com/CATEGORY/SOFTWARE/APPLICATIONS/GAMES/
http://kldp.org/HOWTO/Sound-HOWTO
http://www.radio.org/linux/HAM-HOWTO-14.html
http://www.gamepost.com/old/september98.html
http://www.eiffel-forum.org/archive/category.htm
http://www.sourcecode.se/freelink.htm
http://www.webnix.com/linux/lg/issue35/issue35.html
http://www.webnix.com/linux/lg/issue37/issue37.html
http://www.webnix.com/linux/lg/issue25/issue25.html
http://linuxwww.db.erau.edu/mail_archives/linux-net/Nov_98/
http://linuxcentral.com/linux/lg/issue37/issue37.html
http://linuxcentral.com/linux/lg/issue35/issue35.html
http://linuxcentral.com/linux/lg/issue25/issue25.html
http://www.gl.umbc.edu/~ashinn1/games/
http://www.lib.ox.ac.uk/internet/news/faq/archive/emulators-faq.part1.html
http://www.lib.ox.ac.uk/internet/news/faq/archive/acorn.games.html
http://www.zdwebopedia.com/Data_Storage/hard_card.html
http://ic.net/~jheaton/Bookmark.htm
http://www.linuxresources.com/apps/projects.html
http://tucows.phoenix.net/graze/gr090598.html
http://cdrom.amug.org/http/pilot/games2.html
http://slashdot.org/askslashdot/99/03/07/1352247.shtml
http://slashdot.org/articles/98/10/26/1449214.shtml
http://www.cgallery.com/btow.html
ftp://ftp.the-gc.net/lg/issue35/issue35.html
ftp://ftp.the-gc.net/lg/issue37/issue37.html
http://www.itknowledge.com/reference/library/0789716232/appendix-a.html
http://www.redhat.com/support/docs/rhl/RHL-5.2-Manual/install-guide/manual/doc106.html
http://www.minet.net/docs/lg/issue35/issue35.html
http://www.minet.net/docs/lg/issue37/issue37.html
http://www.xlr8yourmac.com/archives/jan99/011499.html
http://www.xlr8yourmac.com/archives/feb99/020599.html
http://www.xlr8yourmac.com/051498.html
http://www.linuxgazette.com/issue37/issue37.html
http://www.penguincomputing.com/LDP/LDP/LG/issue37/issue37.html
ftp://ftp.patoche.org/lg/issue37/issue37.html
ftp://ftp.patoche.org/lg/issue35/issue35.html
http://www.lg.clight.net/issue37/issue37.html
http://www.lg.clight.net/issue35/issue35.html
http://www.deva.net/lg/issue37/issue37.html
http://www.deva.net/lg/issue35/issue35.html
http://www.patoche.org/lg/issue37/issue37.html
http://www.patoche.org/lg/issue35/issue35.html
http://stommel.tamu.edu/~baum/linuxlist/linuxlist-old/node11.html
http://lg.iafrica.com/issue35/issue35.html
http://lgazette.linuxfr.org/issue37/issue37.html
http://isunix.it.ilstu.edu/resources/lg/issue37/issue37.html
ftp://ftp.uni-erlangen.de/pub/amiga/aminet/disk/INDEX
http://src.doc.ic.ac.uk/public/aminet/info/www/dirs/disk.html
ftp://ftp.unina.it/~aminet/dirs/disk.html
http://www.gnu.org/prep/tasks.html
http://www.slashdot.org/articles/99/01/29/1531205.shtml
http://metalab.unc.edu/metafaq.shtml
http://metalab.unc.edu/pub/Linux/games/multiplayer/!INDEX
http://lwn.net/980226/
http://rufus.w3.org/linux/RPM/redhat/5.1/i386/ByDate.html
http://www.hensa.ac.uk/linux/rpm2html/redhat/5.1/i386/ByDate.html
http://www.monitor.ca/monitor/issues/vol5iss3/lnxstuff.html
http://sunsite.auc.dk/linux/RPM/redhat/5.1/i386/ByDate.html
http://www.decipher.com/help/
http://www.troll.no/qtprogs.html
http://chem4823.usask.ca/linux.html
http://www.3dsoundsurge.com/faqs/vortex1techfaq.html
http://www.cdromshop.com/cdshop/desc/p.743550101291.html
http://www.io.com/~dnk/notebook.html
http://www.nlug.org/smp/
http://www.tcp.ca/gsb/reviews/OS/Linux_versions.html
http://www.cs.berkeley.edu/~manuel/bookmarks.html
http://www.com.univ-mrs.fr/ssc/info/linux/install_linux.html
http://weber.u.washington.edu/~lwinn/
http://www.surfpoint.com/Games--Internet_Games--Multiplayer_Net_Games-1.html
http://darkstar.swsc.k12.ar.us/linux-info.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/!INDEX.short.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/!INDEX.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/libs/ui/!INDEX
http://bugs.gnome.org/db/70/705.html
ftp://ftp.nectec.or.th/pub/mirrors/linux/games/!INDEX.short.html
ftp://ftp.nectec.or.th/pub/mirrors/linux/games/!INDEX.html
ftp://ftp.univie.ac.at/systems/linux/sunsite/games/!INDEX.html
http://www.boutell.com/lsm/lsmsubject.html
http://wkweb3.cableinet.co.uk/linux/LDP/links.html
http://www.freesource.org/lg/issue35/issue35.html
http://www.markom.net/LDP/links.html
http://www.hex.net/~cbbrowne/linuxcommercial.html
http://www.teleport.com/~stevena/scrabble/searches.txt
http://www.ssc.com/lg/issue35/issue35.html
http://www.linuxhq.com/lg/issue35/issue35.html
http://www.pclink.com/tiesma/pinball/rgpsurvy/rgpsurvy.htm
http://www.linuxgazette.com/issue25/issue25.html
http://www.cdromshop.com/cdshop/w95/all.H.html
http://www.cdromshop.com/cdshop/w95/all.C.html
http://www.cdromshop.com/cdshop/desc/p.052145831216.html
http://www.cdromshop.com/cdshop/desc/p.020626702409.html
http://www.cdromshop.com/cdshop/w95/pub/sierra_on-line.html
http://www.cdromshop.com/cdshop/desc/p.767457600176.html
http://www.cdromshop.com/cdshop/ibm/pub/cosmi.html
http://tucows.phoenix.net/graze/gr101397.html
ftp://ftp.the-gc.net/lg/issue25/issue25.html
ftp://ftp.patoche.org/lg/issue25/issue25.html
http://www.linuxhq.com/lg/issue25/issue25.html
http://www.linuxhq.com/v22patch/patch-2.2.4/linux_drivers_net_3c527.c.html
http://www.minet.net/docs/lg/issue25/issue25.html
http://www.freesource.org/lg/issue25/issue25.html
http://www.lg.clight.net/issue25/issue25.html
http://www.deva.net/lg/issue25/issue25.html
http://members.tripod.com/~FWDcomputing/index-5.html
http://www.patoche.org/lg/issue25/issue25.html
http://lg.iafrica.com/issue25/issue25.html
http://lgazette.linuxfr.org/issue25/issue25.html
http://isunix.it.ilstu.edu/resources/lg/issue25/issue25.html
http://isunix.it.ilstu.edu/resources/lg/issue22/issue22.html
http://tucows.exp.net/graze/gr101397.html
http://www.cs.washington.edu/homes/tlau/tome/linux-game.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/libs/ui/!INDEX.short.html
ftp://ftp.infomagic.com/pub/mirrors/linux/sunsite/games/!INDEX
http://www.fokus.gmd.de/linux/linux-softw-misc.html
http://www.hensa.ac.uk/linux/rpm2html/redhat/powertools/5.2/i386/ByDate.html
http://www.hensa.ac.uk/linux/rpm2html/BByName.html
http://www.aisp.net/other.htm
http://www.silug.org/LDP/HOWTO/Quake-HOWTO-2.html
http://slashdot.org/articles/98/12/16/2256243.shtml
http://slashdot.org/articles/99/03/11/2327241.shtml
http://www.independence.seul.org/software/index.distribution.Independence.html
http://www.slashdot.org/articles/98/12/16/2256243.shtml
http://www.gamestorm.com/partners/development.html
http://www.zdnet.com/zdhelp/tip_help/st091698/st091698c.html
http://www.zdnet.com/swlib/pilotsoftware/prespick/0399/ppp0399.html
http://www.cyberfiber.com/
http://www.cyberfiber.com/index.html
http://www.hio.hen.nl/~akkersdi/beheer/rpms.html
http://www.cs.tut.fi/~vespe/bookmarks.html
http://rufus.w3.org/linux/RPM/SGI_Linux.html
http://rufus.w3.org/linux/RPM/Red_Hat_Linux_Colgate.html
http://rufus.w3.org/linux/RPM/Red_Hat_Linux_Vanderbilt.html
http://rpmfind.net/linux/RPM/Red_Hat_Linux_Colgate.html
http://rpmfind.net/linux/RPM/Red_Hat_Linux_Vanderbilt.html
http://www.systemsmgmt.com/archives/9903/5linux28.shtml
http://linuxwww.db.erau.edu/mail_archives/axp-list/Mar_99/
http://www.wired.com/news/news/culture/story/4564.html
http://www.goban.demon.co.uk/bookmark.htm
http://sunsite.auc.dk/linux/RPM/SGIlinux/5.1/ByDate.html
http://sunsite.auc.dk/linux-newbie/FAQ.htm
http://www.hongik.com/linux/linapp5.html
http://www.nada.kth.se/~jespert/pointers.html
http://www.penguincomputing.com/LDP/LDP/gs/node8.html
http://www.linuxlinks.com/Books/
http://www.ispchannel.com/~xgz/linux_game.html
http://features.linuxtoday.com/stories/4659_flat.html
http://linuxtoday.com/stories/4659_flat.html
http://www.linuxmall.com/sofs.html
http://www.linuxmall.com/sofcs.html
http://www.linuxmall.com/sofns.html
http://www.dcs.gla.ac.uk/~gnik/bookmarks.html
http://www8.zdnet.com/swlib/pilotsoftware/prespick/0399/ppp0399.html
http://www.jerrypournelle.com/REPORTS/MOSHE_BAR/bar1.html
http://www.itlibrary.com/reference/library/0672308509/lsgxf.htm
http://lwn.net/1999/0114/bigpage.phtml
http://lwn.net/1999/0204/announce.phtml
http://www.dircsa.org.au/pub/bbslist/bbsfsa.txt
http://www.lib.ox.ac.uk/internet/news/faq/archive/can-sites-newsgroups.part2.html
http://www.visi.com/~nathan/games/linux3d.html
http://www-irm.mathematik.hu-berlin.de/~honig/bookmarks.html
http://www.linux-mandrake.com/userguide/
http://filewatcher.org/file/1761175/
http://www.modeemi.cs.tut.fi/~drdoom/bookmarks.html
http://members.lsol.net/michaelm/linumain.htm
http://www.indelible-blue.com/ibapps/products.nsf/All+Products
http://pertsserver.cs.uiuc.edu/~hull/bookmarks.html
http://www.websysop.com/
http://tile.net/news/s3.html
http://www.linux.lu/
http://www.10mb.com/linux/links/
http://www.hiline.net/users/blakjak/
http://www.macresource.pair.com/mrp/archive/199904.shtml
http://www.via.ecp.fr/lyx/archive/9610/msg00368.html
http://www.cybertown.com/colonies/apartm20.html
http://www.xlr8yourmac.com/archives/jan99/011299.html
http://www.xlr8yourmac.com/archives/apr99/040399.html
http://www.flexion.org/mwave/hack.shtml
http://www.linux.org.il/faq.html
http://www.gamingplace.com/onlinecasinos/atlantisstar.htm
http://ettrich.priv.no/kde.txt
http://www.faqs.org/faqs/acorn/games/
http://www.linuxjournal.com/issue60/lte60more.html
http://vancouver-webpages.com/usenet/ncf.shtml
http://www.zdnet.co.uk/news/1998/22/ns-4672.html
http://www.seul.org/docs/whylinux.html
http://www.xmission.com/~bryang/cmass.html
http://slashdot.org/articles/99/03/27/1422217.shtml
http://slashdot.org/features/99/01/26/1535244.shtml
http://slashdot.org/articles/99/02/01/1434228.shtml
http://slashdot.org/articles/99/02/19/1723241.shtml
http://slashdot.org/articles/99/04/06/1234210.shtml
http://slashdot.org/articles/99/04/08/1345204.shtml
http://www.2600.com/phrack/p51-14.html
http://www.cs.berkeley.edu/~ddgarcia/hotlistNetscape.html
http://www.cdromshop.com/cdshop/ibm/all.C.html
http://www.macresource.com/mrp/archive/199903.shtml
http://www.macresource.com/mrp/archive/199902.shtml
